tests/ui/drop-bounds/drop-bounds-impl-drop.rs

```diff
@@ -2,6 +2,7 @@
 #![deny(drop_bounds)]
 // As a special exemption, `impl Drop` in the return position raises no error.
 // This allows a convenient way to return an unnamed drop guard.
+// Voldemort here refers to types that are unnameable.
 fn voldemort_type() -> impl Drop {
   struct Voldemort;
   impl Drop for Voldemort {
```

Comment:
```suggestion
// As a special exemption, `impl Drop` in the return position raises no error.
// This allows a convenient way to return an unnamed drop guard.
fn unnameable_type() -> impl Drop {
  struct Unnameable;
  impl Drop for Unnameable {
```

Reply:
(and the call right below, but i can't suggest changes on that part of the file :smile_cat:)